docs/docs/03-main-concepts.md

```diff
@@ -82,3 +81,16 @@ Snowpack ships with official support for [webpack](https://www.npmjs.com/package
   "plugins": [["@snowpack/plugin-webpack", {/* ... */}]]
 }
 ```
+
+#### Unbundled optimization (experimental)
+
+To try Snowpack’s unbundled optimization including JS minification and JS [module preloading](https://developers.google.com/web/updates/2017/12/modulepreload), check out `@snowpack/plugin-optimize`. The goal of this plugin is to optimize an unbundled application for production. Note that this is experimental, so the API/plugin options may change as features stabilize.
+
+```js
+// snowpack.config.json
+// [npm install @snowpack/plugin-optimize]
+{
+  // Optimize your unbundled app
+  "plugins": [["@snowpack/plugin-optimize"]]
+}
+```
```

Comment:
@FredKSchott moved this plugin to an external package, for publishing later. Tests still look good, so this seems to be working. WDYT about the doc changes here?

Reply:
overall looks good. Generally I try to keep the code snippets concise, so something like: 

```
// snowpack.config.json
// [npm install @snowpack/plugin-optimize]
{
  "plugins": [
    // Optimize your application
    ["@snowpack/plugin-optimize", {"minifyJS": true, "modulePreload": true}]
  ]
}
```

Reply:
I think we should take a look at naming before publishing anything. The immediate question that comes to mind is "do I use "@snowpack/plugin-optimize" and "@snowpack/plugin-webpack" together?"